tests/unit/test_props_parser.py

```diff
@@ -91,7 +91,7 @@ def default_props_parser(parser, pp, sects, headers):
 
 @pytest.fixture
 def props_parser_empty_conf(parser, pp):
-    return parser(pp, "props_conf", [])
+    return parser(pp, "props_conf", [], props_conf=[])
```

Comment:
changed line does exactly the same thing

Reply:
It is not exactly the same. It is needed to cover line: https://github.com/splunk/pytest-splunk-addon/blob/0750f0fd9fd4e6f56a969cd34fafe754df6b0bdb/pytest_splunk_addon/standard_lib/addon_parser/props_parser.py#L109 In one case we are returning object with empty lists but it is an object. In second case we return empty list so we can go into `if not` line

Reply:
yes, you're right, it differs when we want to get empty list instead of empty configuration_file